### PR TITLE
Added auth page

### PR DESCRIPTION
**Pull request by ziccardi**
Adding authentication page

### CODE REVIEW

File: Example/Example/Views/Pages/UserDetails.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public partial class UserDetails : ContentPage
+    {
+        public UserDetails()
+        {
+            InitializeComponent();
+            initListView();
+        }
+
+        private void initListView() {
+            List<String> roles = new List<string>();
```
Comment (josemigallas) [inline Example/Example/Views/Pages/UserDetails.xaml.cs]: Use list initializer:
```
List<String> roles = new List<string> { "Role1", "Role2", "Role3" };

File: Example/Example/Views/Pages/UserDetails.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public partial class UserDetails : ContentPage
+    {
+        public UserDetails()
+        {
+            InitializeComponent();
+            initListView();
+        }
+
+        private void initListView() {
```
Comment (josemigallas) [inline Example/Example/Views/Pages/UserDetails.xaml.cs]: Methods are usually named using Pascal case

File: Example/Example/Views/Pages/AuthPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class AuthPage : ContentPage
+    {
+        public AuthPage()
+        {
+            InitializeComponent();
+        }
+
+        public void OnAuthenticateClicked(object sender, EventArgs args)
```
Comment (josemigallas) [inline Example/Example/Views/Pages/AuthPage.xaml.cs:17]: Can you cast the argument here directly? `(Button sender, EventArgs args)`

File: Example/Example/Views/Pages/UserDetails.xaml.cs (hunk 2 above)
Comment (ziccardi) [reply Example/Example/Views/Pages/UserDetails.xaml.cs]: Yeah, I'm too java addicted :D

File: Example/Example/Views/Pages/UserDetails.xaml.cs (hunk 1 above)
Comment (ziccardi) [reply Example/Example/Views/Pages/UserDetails.xaml.cs]: 👍

File: Example/Example/Views/Pages/AuthPage.xaml.cs (hunk 3 above)
Comment (ziccardi) [reply Example/Example/Views/Pages/AuthPage.xaml.cs:17]: Nope. A runtime error is thrown if I change the signature.

File: Example/Example.Android/Resources/drawable/ag_background.svg [hunk 4]
```diff
@@ -0,0 +1,5205 @@
+﻿<svg xmlns="http://www.w3.org/2000/svg"
```
Comment (wojta) [inline Example/Example.Android/Resources/drawable/ag_background.svg:1]: Rendering of large SVGs is really slow. We should go with the PNGs.

File: Example/Example.Android/Resources/drawable/ic_phonelink_lock.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```
Comment (wojta) [inline Example/Example.Android/Resources/drawable/ic_phonelink_lock.svg:1]: SVGs are now both in Example and Example.Android. I think it that they should be only in Example.

**Review by wojta — CHANGES_REQUESTED** [review]

**Comment by wojta** [comment]
I'm otherwise ok with the code.

**Comment by ziccardi** [comment]
@wojta Mind giving another look?

**Review by wojta — APPROVED** [review]

**Pull request merged**